### PR TITLE
Jango/minor improvements

### PR DESCRIPTION
**Pull request by Bigjango13**
Adds a few minor things that we discussed, along with fixing a few typos and mistakes.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -691,6 +716,13 @@ Args will also return a list of arguments passed to the program if called outsid
 print ( args() );
 ```
 If the function that is called has no arguments, `args()` will return `none`, however if args is called in the global scope it's length will always be greater than 0.
+The `rand()` function can be used to get a random number between 0 and 1 (0 included), for example:
+```
+let e = rand();
+# Nobody knows exactly what e is a now
+# But it's know that type(e) == "Decimal"
```
Comment (nobody5050) [inline readme.md]: known*

**Review by nobody5050 — CHANGES_REQUESTED** [review]
Looks good except for this typo

**Comment by nobody5050** [comment]
Oh also, following up on what lumin had suggested; making things return none instead of crashing in some instances

**Comment by Bigjango13** [comment]
> Oh also, following up on what lumin had suggested; making things return none instead of crashing in some instances

I'm not sure which parts you are referring to, there are many ways to cause runtime errors.

**Comment by nobody5050** [comment]
Specifically what lumin mentioned, accessing a variable that doesn’t exist

**Review by nobody5050 — APPROVED** [review]

**Pull request merged**